manuscript/recipes/duplicati.md

```diff
@@ -49,16 +41,16 @@ cd /var/data/config/duplicati
 
 1. Generate a random passphrase to use to encrypt your data. **Save this somewhere safe**, without it you won't be able to restore!
 2. Seriously, **save**. **it**. **somewhere**. **safe**.
-3. Create duplicati.env, and populate with the following variables (with your appropriate time zone)
+3. Create `duplicati.env`, and populate with the following variables (replace "Europe/London" with your appropriate time zone from [this list](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones))
```

Comment:
Use italics for any text in brackets (improves readability of the Ubuntu font in large blocks of text)

Reply:
Fixed